hs_core/models.py

```diff
@@ -1185,9 +1186,8 @@ def bag_url(cls, resource_id):
     @classmethod
     def scimeta_url(cls, resource_id):
         scimeta_path = "{resource_id}/data/resourcemetadata.xml".format(resource_id=resource_id)
-        istorage = IrodsStorage()
-        scimeta_url = istorage.url(scimeta_path)
-
+        scimeta_url = reverse('rest_download',
+                          kwargs={'path': scimeta_path})
```

Comment:
@hyi I have resolved the import of 'receivers' for timeseries resource. You can look at that resource code. Otherwise, we can hipchat tomorrow on this.